packages/server/postgres/utils/checkEqBase.ts

```diff
@@ -1,7 +1,18 @@
+import areDeep from 'fbjs/lib/areEqual'
 import {RTable, TableSchema} from '../../database/stricterR'
-import areEqual from 'fbjs/lib/areEqual'
 import getPg from '../getPg'
 
+function areEqualIgnoringOrder(a, b): boolean {
+  if (Array.isArray(a) && Array.isArray(b)) {
+    return (
+      a.length === b.length &&
+      a.every((aa) => b.some((bb) => areDeep(aa, bb))) &&
+      b.every((bb) => a.some((aa) => areDeep(aa, bb)))
```

Comment:
Not sure if we need to tweak the optimization here (O(n^2) -> O(n) by tracking visited items and etc) and complicate the code.  It depends on the amount of data the system has to process.